lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -20,42 +20,16 @@ def initialize(options = {})
 
       def purchase(money, payment, options = {})
         post = {}
-        add_invoice(post, money, options)
-        add_payment(post, payment, options)
-        add_3ds(post, payment, options)
-        add_address(post, payment, options)
-        add_metadata(post, options)
-        add_ewallet(post, options)
-        add_payment_fields(post, options)
-        add_payment_urls(post, options)
-        add_customer_id(post, options)
+        add_auth_purchase(post, money, payment, options)
         post[:capture] = true if payment.is_a?(CreditCard)
```

Comment:
The only thing I am wondering here is whether folks will be transacting on Rapyd with any other tokenized payment method type that is not a Check and not a CreditCard, and whether we might want `capture: true` in those cases?